doc/classes/CubeMap.xml

```diff
@@ -45,6 +49,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Return the width of the CubeMap.
```

Comment:
- Return -> Returns
- Per the writing guidelines, use possessive:

Returns the CubeMap's width.